fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -239,6 +240,36 @@ public String getObjectsAsString(final Graph graph,
         return "";
     }
 
+    /**
+     * Returns the original resource as a URI Node if
+     * the subject represents a memento uri; otherwise it
+     * returns the subject parameter.
+     * @param subject the subject
+     * @return a URI node of the original resource.
+     */
+    public Node getOriginalResource(final Node subject) {
+        if (!subject.isURI()) {
+            return subject;
+        }
+
+        final String subjectUri = subject.getURI();
+        final int index = subjectUri.indexOf(FCR_VERSIONS);
+        if (index > 0) {
+            return NodeFactory.createURI(subjectUri.substring(0, index - 1));
+        } else {
+            return subject;
+        }
+    }
+
+    /**
+     * Same as above but takes a string.
+     * @param subjectUri the URI
+     * @return a node
+     */
+    public Node getOriginalResource(final String subjectUri) {
```

Comment:
This looks good and works well. 
One minor suggestion, since it appears that this method is not used (and may in the future be at risk of being deleted) please add a comment indicating that the method is used in `fcrepo-http-api/src/main/resources/views/default.vsl`.